src/Web/Sfa.Das.Sas.Web.CloudService/ServiceConfiguration.Local.cscfg

```diff
@@ -21,7 +21,7 @@
       <Setting name="ElasticsearchUsername" value="" />
       <Setting name="ElasticsearchPassword" value="" />
       <Setting name="FeatureToggle.IgnoreSslCertificateFeature" value="" />
-      <Setting name="FeatureToggle.Elk5Feature" value="" />
+      <Setting name="FeatureToggle.Elk5Feature" value="true" />
```

Comment:
I noticed that there's a PR outstanding in das-search for removing some of the redundant stuff like this feature flag.